app/models/group.rb

```diff
@@ -12,22 +12,20 @@ class Group < ApplicationRecord
   private
 
   def start_date_after_end_date
-    if start_date.blank? || end_date.blank?
-      return
-    end
-
-    if end_date <= start_date
-      t(".group.new.start_date_after_end_date_error")
+    if start_date.present? && end_date.present? && (end_date <= start_date)
```

Comment:
Could this line change to: `if start_date && end_date && (end_date <= start_date)`?

Reply:
Are there specs for this validation?

Reply:
Yes there is a spec for testing `start_date_in_the_past` in the model/group_spec.rb. I just added the spec for the `start_date_after_end_date`.

Done in sha a80758b

I also removed the `present?` method in sha 83fd409